test/e2e/common_test.go

```diff
@@ -306,6 +299,12 @@ func PodmanTestCreateUtil(tempDir string, remote bool) *PodmanTestIntegration {
 		}
 	}
 
+	INTEGRATION_ROOT = os.Getenv("PODMAN_INTEGRATION_ROOT")
+	if INTEGRATION_ROOT == "" {
+		cwd, _ := os.Getwd()
+		INTEGRATION_ROOT = filepath.Join(cwd, "../")
+	}
```

Comment:
this should be set once per node in the second function of SynchronizedBeforeSuite() instead of doing the same logic for every single test

Reply:
I totally agree, except, shouldn't that apply to *all* the settings here in this block also? It's not likely that `$PODMAN_REMOTE_BINARY`, `QUADLET`, `OCI_RUNTIME`, etc will change? May I leave that for a future cleanup?

Reply:
I think it is better to leave this for a future cleanup